gson/src/test/java/com/google/gson/JsonObjectTest.java

```diff
@@ -92,6 +92,7 @@ public void testAddingStringProperties() throws Exception {
     assertEquals(value, jsonElement.getAsString());
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
This is for `jsonElement.getAsCharacter()`, right? Perhaps we could do the same thing of putting it in a local and putting the `@SuppressWarnings` on the declaration of the local?